.github/workflows/build.yml

```diff
@@ -7,18 +7,21 @@ on:
   workflow_dispatch:
   workflow_call:
 
+env:
+  CARGO_TERM_COLOR: always
+
 jobs:      
   test_and_build_linux_amd64:
     runs-on: ubuntu-latest
     container:
-      image: ghcr.io/eclipse-ankaios/devcontainer-base:0.6.2
+      image: ghcr.io/eclipse-ankaios/devcontainer-base:0.6.3
       options: --privileged
     steps:
       - uses: actions/checkout@v3.5.3
       - uses: Swatinem/rust-cache@v2.6.2
       
       - name: Run tests
-        run: RUST_LOG=debug cargo test -- --show-output --nocapture
+        run: RUST_LOG=debug cargo nextest run
```

Comment:
Does the new test exec lib include the prints/logs by default or why is "nocapture" removed now?

Reply:
You will get printf/logs outputs when only when a test failed, so you don't need to "pollute" the output with debug logs. Here an example output: 
![image](https://github.com/eclipse-ankaios/ankaios/assets/42992756/3fb25f2f-03fb-44f4-b345-577e54cdf82b)

Reply:
Ok I agree.